plugins/net_plugin/net_plugin.cpp

```diff
@@ -2000,22 +2001,26 @@ namespace eosio {
    {
    }
 
-   uint32_t sync_manager::active_sync_fetch_span() const {
+   uint32_t sync_manager::active_sync_fetch_span(bool log) const {
       const uint32_t constrained_reversible_remaining = [&]() -> uint32_t {
          const int32_t reversible_remaining = my_impl->chain_plug->chain().max_reversible_blocks_allowed();
```

Comment:
I thought we were not going to use `conf.max_reversible_blocks` anymore when considering `sync_fetch_span`.

This seems clearly incorrect since when `conf.max_reversible_blocks`  is set to 0, `max_reversible_blocks_allowed()` returns `std::numeric_limits<int32_t>::max()` and we don't want `sync_fetch_span` to be infinite.

Reply:
This PR was created before that decision. It is removed in the PR that builds on this one: https://github.com/AntelopeIO/spring/pull/652

Reply:
Never mind, I see you did that in the other PR.